Traceback (most recent call last):
  File "/private/groups/brookslab/gabai/projects/Add-seq/scripts/makeArrow.py", line 1, in <module>
    from modTool import alignScore, modPredict
  File "/private/groups/brookslab/gabai/projects/Add-seq/scripts/modTool.py", line 39, in <module>
    model.load_state_dict(torch.load(weights_path, map_location=torch.device(device)))
  File "/private/groups/brookslab/gabai/miniconda3/envs/addseq/lib/python3.9/site-packages/torch/serialization.py", line 699, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/private/groups/brookslab/gabai/miniconda3/envs/addseq/lib/python3.9/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/private/groups/brookslab/gabai/miniconda3/envs/addseq/lib/python3.9/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './nanopore_classification/best_models/addseq_resnet1d.pt'
